### PR TITLE
Error page js cleaning and clarifying

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
As usual in this project, the *ErrorPage.cs files are generated.

Unfortunately all the functions here were used other than toggleClass.

### CODE REVIEW

File: samples/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*/Properties
```
Comment (NTaylorMullen) [inline samples/.gitignore]: ?

**Comment by NTaylorMullen** [comment]
Are these used: https://github.com/aspnet/Diagnostics/blob/205f5a6ffb96f85d5c91574a6811d3780db4b404/src/Microsoft.AspNet.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.js#L19-L23

File: src/Microsoft.AspNet.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿(function (window, undefined) {
     "use strict";
 
-    function $(selector, element) {
+    function NS(selector, element) {
```
Comment (NTaylorMullen) [inline src/Microsoft.AspNet.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.js]: `ns` (javascript convention)

**Comment by pranavkm** [comment]
I was kinda hoping this would remove all the fluent ness from the code. Plus we should be fine dropping IE8 (maybe IE9) support too. :watch:

**Comment by pranavkm** [comment]
Something along these lines: https://gist.github.com/pranavkm/f78e50e7c61ef7d3f8d2

File: samples/.gitignore (hunk 1 above)
Comment (ryanbrandenburg) [inline samples/.gitignore]: Running the samples caused them to create property folders, which showed up in git status. That kind of thing drives me bonkers, so I hid them.

**Comment by ryanbrandenburg** [comment]
As per in person conversations with Eilon we're sticking with the current fluent-ness and just trimming it down.

**Comment by ryanbrandenburg** [comment]
:up: :date:

File: samples/.gitignore (hunk 1 above)
Comment (NTaylorMullen) [inline samples/.gitignore]: Odd, do you see the same thing when you run the MVC Samples? We don't have this restriction there.

Comment (Eilon) [inline samples/.gitignore]: Could probably revert this for now. We can always add it back later if it seems more definitely needed.

**Comment by Eilon** [comment]
I think just revert the ignore file, then :shipit:

**Comment by NTaylorMullen** [comment]
:shipit: once you've addressed @Eilon's comments.

**Pull request merged**